docs/getting_started/licensing.md

```diff
@@ -1,9 +1,70 @@
+- **Batch:** License type that allows the user to run Shift from a batch script or terminal without the ability to use the graphical interface for manipulating workflows. This licensing type is intended for users that want to run Shift from a terminal or in a Python script to execute a workflow previously setup using Shift with an Interactive License.
+
+To be able to activate Shift it is required to purchase a `PRODUCT KEY`. Product keys can be purchased through Inbibo's official [website](https://inbibo.co.uk/shift#prices). Shift product keys have the following characteristics:
+
+- A `PRODUCT KEY` is associated to one single license type: Interactive or Batch. If the user wants to use Shift both in Interactive and Batch mode, two separate product keys have to be purchased.
+- A product key consists of **28** alphanumeric characters separated by "-" which is provided to the user when purchasing Shift through the website: `XXXX-XXXX-XXXX-XXXX-XXXX-XXXX-XXXX`.
+
 ## Node-Locked Licensing
 
-To Be Defined
+Node-Locked Licensing in Shift requires the activation of a `PRODUCT KEY` on one single machine. As commented before, this activation process requires the user to activate the product for batch and interactive modes separately.
+
+### Interactive
+
+Whenever activating Shift for the first time, a dialog will be prompted requesting to enter a valid `PRODUCT KEY` or to launch Shift in trial mode for non-commercial purposes.
+
+To activate Shift in Node-Locked Interactive mode:
+
+  1. Launch Shift by clicking on the shift.bat file in Shift's installation folder.
+  3. A dialog will show up where to enter a valid `PRODUCT KEY` in order to enable the full license; or to continue with the 30 day trial period.
+
+<figure>
+      <img src="images/license_activation_dialog.png" alt="Shift License Activation Dialog">
+      <figcaption><b>Figure 1</b>: Activation Dialog.</figcaption>
+</figure>
+
+  3. On the text box, enter the `PRODUCT KEY` associated to your Interactive Node-Locked License.
+  4. Click on *Activate* on the bottom left corner.
+
+If the product key is invalid or connecting with the licensing servers failed, an error will be raised indicating that Shift couldn't be activated. Otherwise, Shift will be open, meaning that the license was activated successfully.
+
+> [!NOTE]
+> - This activation mode requires access to the internet for activating licenses.
+> - For deactivating licenses to switch to a different machine, please contact support.
+
+### Batch
+
+Whenever activating Shift for the first time in batch mode, a product key has to be registered previously. Activating batch mode requires the use of an executable that eases the activation process of the product.
```

Comment:
```suggestion
Whenever using Shift for the first time in batch mode, a product key has to be registered first.
```

Reply:
But it does need the batch_activation script right? Why removing the second sentence?

Reply:
Because it is a utility script we wrote. The user technically has no other way to activate a batch license atm.
The script is called batchActivate.py and it is inside the "tools" folder.

My suggestion was meant to reformulate and simplify the sentence.
Activation of shift and registration of the license key are the same thing. Am I wrong? @Inbibo/shift

Reply:
Right. I can simplify the first sentence.
I'm just having trouble understanding why the second sentence should be omitted. The script is in place to make it easier for the user to activate the batch license, and that's exactly what it's stating. If there's no alternative way to do it, I don't get why that information needs to be removed.

Reply:
```suggestion
To activate a Shift Batch license it is required to execute the provided Python script and enter the acquired product key. 
```